packages/insomnia-components/components/sidebar/sidebar-info.js

```diff
@@ -28,7 +28,7 @@ function SidebarInfo(props: Props) {
           <SidebarTextItem label={'Version:'} headline={version} />
         </SidebarItem>
       )}
-      {license.name && (
+      {license && license.name && (
```

Comment:
```suggestion
      {license?.name && (
```

[Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) is a friend 🤗

Reply:
Tried enabling in flow via esproposal.optional_chaining=enable within components and restarted flow server but no luck. It has already been enabled in app. Not sure why it isn't happy here, I'll circle back on this.

Reply:
You might need to install `@babel/plugin-proposal-optional-chaining` and add that to `.babelrc`.

https://github.com/Kong/insomnia/blob/93c91ebdbed9dc635056c920686913722fd19965/packages/insomnia-app/.babelrc#L14